app/controllers/register_controller.rb

```diff
@@ -63,20 +63,50 @@ def create
 
     payload = generate_canonical_object(fields, params)
 
-    @change = Change.new(register_name: params[:register], payload: payload, user_id: current_user.id)
-    @change.status = Status.new(status: 'pending')
-    @change.save
+    if current_user.basic?
+      @change = Change.new(register_name: params[:register], payload: payload, user_id: current_user.id)
```

Comment:
Since this `Change.new` is the same for both user types. We could make it a lambda? e.g.
`create_change = lambda { Change.new(register_name: params[:register], payload: payload, user_id: current_user.id) }`

`@change = create_change.call`

Reply:
I don't we think we should be doing this at all here. We should be using proper resources i.e. `foo.com/country/AU/change/new` or `foo.com/country/new`